### PR TITLE
Added option to to output Lower Hybrid Current Drive in plot_proc

### PR DESCRIPTION
**Pull request by ym1906**
Also added Ion Cyclotron Current Drive support. 

Removed option 9 from plot_proc as it has been removed from PROCESS.

### CODE REVIEW

File: process/io/plot_proc.py [hunk 1]
```diff
@@ -2196,6 +2199,54 @@ def plot_current_drive_info(axis, mfile_data, scan):
...
             data.insert(2, (pinjie, "Total auxillary power", "MW"))
 
+    if lhcd:
+        data = [
+            (pinjie, "Steady state auxiliary power", "MW"),
+            ("pheat", "Power for heating only", "MW"),
+            ("bootipf", "Bootstrap fraction", ""),
+            ("faccd", "Auxiliary fraction", ""),
+            ("facoh", "Inductive fraction", ""),
+            ("powerht", "Plasma heating used for H factor", "MW"),
```
Comment (ajpearcey) [inline process/io/plot_proc.py:2209]: in the NBI output `gamnb` is printed. We should include the normalised current drive efficiency in the other outputs. Can we add a line for `gamcd` here? Also add this in the ERCH and EBW cases.

File: process/io/plot_proc.py [hunk 2]
```diff
@@ -2196,6 +2199,54 @@ def plot_current_drive_info(axis, mfile_data, scan):
...
+            data.insert(2, (pinjie, "Total auxillary power", "MW"))
+
+    if iccd:
+        data = [
+            (pinjie, "Steady state auxiliary power", "MW"),
+            ("pheat", "Power for heating only", "MW"),
+            ("bootipf", "Bootstrap fraction", ""),
+            ("faccd", "Auxiliary fraction", ""),
+            ("facoh", "Inductive fraction", ""),
+            ("powerht", "Plasma heating used for H factor", "MW"),
```
Comment (ajpearcey) [inline process/io/plot_proc.py:2233]: add `gamcd` here

File: process/io/plot_proc.py (hunk 1 above)
Comment (ym1906) [reply process/io/plot_proc.py:2217]: I added this in and also for ECRH and EBW

**Review by ajpearcey — APPROVED** [review]
LGTM!

**Pull request merged**